src/shogun/converter/ica/FastICA.cpp

```diff
@@ -128,9 +128,9 @@ CFeatures* CFastICA::apply(CFeatures* features)
 
 	W = sym_decorrelation(W);
 
-	int iter = 0;
 	float64_t lim = tol+1;
-	while (lim > tol && iter < max_iter)
+	for (auto i : progress(
+	         range(0, max_iter), "PROGRESS: ", UTF8, [&] { return lim > tol; }))
```

Comment:
Minor, for consistency, indent and break lines like in the doc reference you've cited.

Reply:
Thanks @iglesias . The style has been modified.